R/make_plots.R

```diff
@@ -247,8 +250,14 @@ plot_barchart_by_tiers <- function(data, display_type, bar_type){
             Tier = get_tier_name(variable),
             value = value*af_conversion/1000)
    lab_str <- "Usage During Time Period (Thousand AF)"
- }
+  }
 
+  # Get tier labels and colors in the proper order
+  tiernames <- unique(d$Tier)
+  tiernames <- tiernames[order(tiernames)]
```

Comment:
Are `tiernames` always going to be numbers?

Reply:
`tiernames` is a vector of strings ("Tier1", "Tier2", ...)
`order()` returns indices in sorted order, so that is always integers.